specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/examples/ApiManagementCreateKubernetesWorkspace.json

```diff
@@ -0,0 +1,51 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2024-10-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "workspaceId": "wks1",
+    "kind": "KubernetesManaged",
```

Comment:
This is readonly so we should remove it

Reply:
Removed Kind, which I believe is the one that was readonly.